example/otlp/go.mod

```diff
@@ -0,0 +1,8 @@
+module go.opentelemetry.io/otel/example/otel-test
+
+go 1.13
+
```

Comment:
This needs a `replace` directive for the `otel` module so that it functions correctly in CI.  See [here](https://github.com/open-telemetry/opentelemetry-go/blob/master/example/grpc/go.mod#L5).

Reply:
`replace` added